### PR TITLE
{bio}[intel/2018a] PRC v1.5.6

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PRC/PRC-1.5.6-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ profile-sequence comparisons. PRC takes into account all transition and emission probabilities in both
+ hidden Markov models. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://supfam.org/PRC/programs/PRC-%(version)s/']
+sources = ['%(namelower)s-%(version)s-src.tgz']
+checksums = ['2ad9dcd0621a7262b5754426a62cd630dfbf14b04481fde116fa169d89249394']
+
+postinstallcmds = ["mkdir -p %(installdir)s/bin && cp prc convert_to_prc merge_aligns.pl %(installdir)s/bin"]
```
Comment (boegel) [inline easybuild/easyconfigs/p/PRC/PRC-1.5.6-intel-2018a.eb]: @smoors So `make install` doesn't do anything?

Comment (smoors) [reply easybuild/easyconfigs/p/PRC/PRC-1.5.6-intel-2018a.eb]: no, it's not even defined in the makefile

Comment (boegel) [reply easybuild/easyconfigs/p/PRC/PRC-1.5.6-intel-2018a.eb]: @smoors Then we should skip the `install` step (and maybe include a comment to explain why we're using this approach):

```python
skipsteps = ['install']
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/65c6fcb614cbb59b1ddba62a0a7d0dc7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2533.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9406858821a89e281e802338ef6ab850 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**